src/main/java/net/coderbot/iris/mixin/MixinGameRenderer.java

```diff
@@ -34,4 +34,11 @@
 	private boolean disableVanillaHandRendering(GameRenderer gameRenderer) {
 		return !Iris.getCurrentPack().isPresent();
 	}
+	
+	@Inject(method = "getNightVisionScale", at = @At(value = "INVOKE", target = "Lnet/minecraft/world/effect/MobEffectInstance;getDuration()I"), cancellable = true)
+	private static void iris$safecheckNightvisionStrength(LivingEntity livingEntity, float f, CallbackInfoReturnable<Float> cir){
+		if(livingEntity.getEffect(MobEffects.NIGHT_VISION) == null){
```

Comment:
This injection could apply before Origins, and might actually break the effect in the mod completely.